### PR TITLE
And/temporal payment notifications

### PR DESCRIPTION
**Pull request by johanv26**
## Description

Describe what this pull request changes, and why. Include implications for people using this change.

Useful information to include:
- Include screenshots for changes to the UI (ideally, both "before" and "after" screenshots, if applicable).
- Provide links to the description of corresponding configuration changes. Remember to correctly annotate these
changes.

## Additional information

Include anything else that will help reviewers and consumers understand the change.
- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, security, or accessibility.
- Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.

## Checklist for Merge

- [ ] Tested in a remote environment
- [ ] Updated documentation
- [ ] Rebased master/main
- [ ] Squashed commits

### CODE REVIEW

File: eox_nelp/payment_notifications/admin.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+"""
+...
```
Comment (johanv26) [inline eox_nelp/payment_notifications/admin.py:2]: Add more explanation for the description

File: eox_nelp/payment_notifications/context_processor.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
 """
+...
```
Comment (johanv26) [inline eox_nelp/payment_notifications/context_processor.py:2]: Some description

File: eox_nelp/payment_notifications/models.py [hunk 3]
```diff
@@ -43,18 +43,48 @@ class PaymentNotification(models.Model):
...
-    call_to_action_2_msg = models.CharField(max_length=1000, null=True, blank=True, help_text="Name of the button for case 2")
-    call_to_action_3_msg = models.CharField(max_length=1000, null=True, blank=True, help_text="Name of the button at the end of the message")
-
-    call_to_action_1_url = models.CharField(max_length=1000, null=True, blank=True, help_text="Link of the button for case 1")
-    call_to_action_2_url = models.CharField(max_length=1000, null=True, blank=True, help_text="Link of the button for case 2")
-    call_to_action_3_url = models.CharField(max_length=1000, null=True, blank=True, help_text="Link of the button at the end of the message")
+    custom_msg = models.TextField(
+        null=True,
+        blank=True,
+        help_text='Use a "lang-ar" or "lang-en" class to select language')
```
Comment (johanv26) [inline eox_nelp/payment_notifications/models.py:56]: The close parenthesis could go in new line `)`

File: eox_nelp/settings/common.py [hunk 4]
```diff
@@ -60,9 +60,9 @@ def plugin_settings(settings):
     if find_spec('eox_audit_model') and EOX_AUDIT_MODEL_APP not in settings.INSTALLED_APPS:
         settings.INSTALLED_APPS.append(EOX_AUDIT_MODEL_APP)
 
-
     try:
-        payments_notifications_context = 'eox_nelp.payment_notifications.context_processor.payments_notifications_context'
+        payments_notifications_context = 'eox_nelp.payment_notifications.context_processor.payments_notifications_context'  # pylint: disable=line-too-long
```
Comment (johanv26) [inline eox_nelp/settings/common.py]: Does this really need a  skip of `line-too-long` or could be split into more lines?

File: eox_nelp/settings/common.py [hunk 5]
```diff
@@ -61,7 +61,9 @@ def plugin_settings(settings):
         settings.INSTALLED_APPS.append(EOX_AUDIT_MODEL_APP)
 
     try:
-        payments_notifications_context = 'eox_nelp.payment_notifications.context_processor.payments_notifications_context'  # pylint: disable=line-too-long  # noqa: E501
+        payments_notifications_context = (
+            'eox_nelp.payment_notifications.'
+            'context_processor.payments_notifications_context')
```
Comment (johanv26) [inline eox_nelp/settings/common.py:66]: The close parenthesis could go in new line )

**Comment by Ever3tt** [comment]
Hola @andrey-canon, ya se solucionaron los errores del PR, asi vamos

Screenshot from 2023-10-02 16-09-34

image

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**